src/Symfony/Component/Notifier/Recipient/SmsRecipientTrait.php

```diff
@@ -16,18 +16,9 @@
  *
  * @experimental in 5.1
  */
-class SmsRecipient implements SmsRecipientInterface
+trait SmsRecipientTrait
 {
-    use PhoneValidationTrait;
-
-    private $phone;
-
-    public function __construct(string $phone)
-    {
-        self::ensurePhoneIsValid($phone);
-
-        $this->phone = $phone;
-    }
+    protected $phone;
```

Comment:
Not sure if this should be protected. As the Recipient class is not final it might be useful when extending it.